### PR TITLE
[plugin.video.playonbrowser] v1.0.20

### PR DESCRIPTION
**Pull request by Lunatixz**
### Description
Jarvis+ update, prior version found in helix branch.

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by enen92** [comment]
Do you plan also to remove the version found in the helix branch?

File: plugin.video.playonbrowser/default.py [hunk 1]
```diff
@@ -0,0 +1,845 @@
...
+if sys.version_info < (2, 7):
+    import simplejson as json
+else:
+    import json
+   
+## GLOBALS ##
+
+# Plugin Info
+ADDON_ID = 'plugin.video.playonbrowser'
+REAL_SETTINGS = xbmcaddon.Addon(id=ADDON_ID)
```
Comment (enen92) [inline plugin.video.playonbrowser/default.py]: no need for the id here

File: plugin.video.playonbrowser/default.py [hunk 2]
```diff
@@ -0,0 +1,845 @@
...
+ADDON_ID = REAL_SETTINGS.getAddonInfo('id')
+ADDON_NAME = REAL_SETTINGS.getAddonInfo('name')
+ADDON_PATH = REAL_SETTINGS.getAddonInfo('path').decode('utf-8')
+ADDON_VERSION = REAL_SETTINGS.getAddonInfo('version')
+ICON = os.path.join(ADDON_PATH, 'icon.png')
+FANART = os.path.join(ADDON_PATH, 'fanart.jpg')
+LANGUAGE = REAL_SETTINGS.getLocalizedString
+
+# Playon Info
+MEDIA_PATH = ADDON_PATH + '/resources/media/'
```
Comment (enen92) [inline plugin.video.playonbrowser/default.py]: can you please use the `os` module to concatenate the path?

File: plugin.video.playonbrowser/default.py [hunk 3]
```diff
@@ -0,0 +1,845 @@
...
+PLAYON_ICON = '/images/play_720.png'
+PLAYON_DATA = '/data/data.xml'
+BASE_URL = sys.argv[0]
+BASE_HANDLE = int(sys.argv[1])
+args = urlparse.parse_qs(sys.argv[2][1:])
+REAL_SETTINGS = xbmcaddon.Addon(id=ADDON_ID)
```
Comment (enen92) [inline plugin.video.playonbrowser/default.py]: repeated

**Comment by Lunatixz** [comment]
> Do you plan also to remove the version found in the helix branch?

Yes, the helix plugin should be removed. I'll resubmit those corrects Thanks.

**Comment by Lunatixz** [comment]
Can't squash commit any ideas?

**Comment by enen92** [comment]
we can squash them when merging

**Comment by enen92** [comment]
Can you create a separate pull request to remove the plugin from the helix repository?

File: plugin.video.playonbrowser/addon.xml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        <import addon="script.module.simplejson" version="3.3.0"/>
+        <import addon="script.module.simplecache" version="1.0.0"/>
+        <import addon="script.module.metahandler" version="2.9.0"/>
+        <import addon="script.module.skin.helper.artutils" version="1.0.0" />
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="default.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_US">Allows you to browse m.playon.tv from Kodi.</summary>
```
Comment (enen92) [inline plugin.video.playonbrowser/addon.xml]: by default is en_GB. Can you change the language tags (or at least replicate the provided ones)?

File: plugin.video.playonbrowser/resources/language/English/strings.po [hunk 5]
```diff
@@ -0,0 +1,78 @@
+# Playon Browser Media Center language file
```
Comment (enen92) [inline plugin.video.playonbrowser/resources/language/English/strings.po]: Also, something I forgot. Can you change the folder layout as well? See here: https://github.com/xbmc/generator-kodi-addon/tree/master/generators/app/templates/resources/language

Comment (Lunatixz) [inline plugin.video.playonbrowser/resources/language/English/strings.po]: ah, I didn't catch that format change... thx for the info, I'll make the changes.

**Comment by enen92** [comment]
Thanks

**Pull request merged**